datafusion/sqllogictest/test_files/timestamps.slt

```diff
@@ -2858,7 +2858,7 @@ statement error
 select to_local_time('2024-04-01T00:00:20Z'::timestamp, 'some string');
 
 # invalid argument data type
-statement error DataFusion error: Execution error: The to_local_time function can only accept timestamp as the arg, got Utf8
+statement error DataFusion error: Error during planning: Execution error: User\-defined coercion failed with Plan
```

Comment:
I think it is better to have the reason of error `The to_local_time function can only accept timestamp as the arg, got Utf8`

Reply:
Done!